auslib/blobs/gmp.py

```diff
@@ -11,6 +11,19 @@ def __init__(self, **kwargs):
         if "schema_version" not in self:
             self["schema_version"] = 1000
 
+    def validate(self, product, whitelistedDomains, checkHash=True):
+        Blob.validate(self, product, whitelistedDomains)
+        if checkHash:
+            for vendor in self["vendors"]:
+                if "platform" in self["vendors"][vendor]:
+                    for platform in self["vendors"][vendor]["platforms"]:
```

Comment:
You can probably make this a bit more compact by using .get(). Eg, this could replace line 18 and 19:
`for platform in self["vendors"][vendor].get("platforms", {}).values():`